planner/core/find_best_task.go

```diff
@@ -514,7 +517,7 @@ func (ds *DataSource) convertToPartialIndexScan(prop *property.PhysicalProperty,
 	isCovered bool) {
 	idx := path.index
 	is, partialCost, rowCount := ds.getOriginalPhysicalIndexScan(prop, path, false, false)
-	rowSize := is.indexScanRowSize(idx, ds)
+	rowSize := is.indexScanRowSize(idx, ds, false)
```

Comment:
Why `false`? I thought it should always be `true`.

Reply:
When the row size contributes to the cost of the network. It's `false`. When it contributes to scan cost. It's `true`.

Reply:
Yes. TiDB always receives chunk data.